exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how is the level of complexity of a given text assessed, and how can we make programs which do that? 
+In this exercise, you will write a program which will determine how difficult a text is to read
+and what age it is best suited for.
+
+We will use the
+[Automated readability index (ARI)][wiki] formula for calculating the text complexity.
+The ARI was developed in 1967 and assesses the U.S. grade level required to read 
+a piece of text. In some ways, it is similar to other formulas. Its main difference 
```

Comment:
> In some ways, it is similar to other formulas.

Does this sentence add anything? Can it be dropped?

Reply:
Some exercises have two files: `introduction.md` and `instruction.md`. I think this text fragment may be excluded from instruction, and if we would like to create an introduction, it can be added here. It is a quote from an article.

Reply:
> In some ways, it is similar to other formulas. Its main difference is rather than counting syllables, it counts characters.

can simply be:

> Its main difference from other formulas is rather than counting syllables, it counts characters.

Reply:
fixed